interop/message.go

```diff
@@ -25,6 +26,7 @@ type L2ToL2Message struct {
 	Sender      common.Address
 	Target      common.Address
 	Message     []byte
+	Calldata    []byte
```

Comment:
This shouldn't be apart of the L2ToL2Message struct -- it's a higher level of abstraction from the interop serialization of an event

Reply:
removed